lib/mongo/scrollable.rb

```diff
@@ -41,11 +41,14 @@ def scroll(cursor_or_type = nil, options = nil, &_block)
       # scroll
       if block_given?
         previous_cursor = nil
+        current_cursor = nil
         records.each do |record|
           previous_cursor ||= cursor_type.from_record(record, cursor_options.merge(type: :previous))
+          current_cursor ||= cursor_type.from_record(record, cursor_options.merge(include_current: true))
```

Comment:
`include_current` is not used anywhere? remove

Reply:
It is actually used. If you loop over records 1 to 5, the cursor is built with the record 1 and this record have to be included if you want to do the same loop.